packages/babel-plugin-transform-modules-systemjs/test/fixtures/systemjs/hoisting-bindings/output.mjs

```diff
@@ -4,8 +4,10 @@ System.register([], function (_export, _context) {
   var c;
 
   function a() {
+    var _c;
+
     alert("a");
-    _export("c", c + 1), c++;
+    _export("c", _c = c++), _c;
```

Comment:
This is wrong, it should export c's new value, not the old one.

Reply:
Whoops.